sources/packages/backend/workflow/test/2023-2024/fulltime-assessment/eligibility/fulltime-assessment-2023-2024-eligibility-CSGD.e2e-spec.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  createFakeConsolidatedFulltimeData,
+  executeFulltimeAssessmentForProgramYear,
+} from "../../../test-utils";
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  DependentEligibility,
+  createFakeStudentDependentEligible,
+  createFakeStudentDependentNotEligible,
+} from "../../../test-utils/factories";
+
+describe(`E2E Test Workflow fulltime-assessment-${PROGRAM_YEAR}-eligibility-CSGD.`, () => {
+  it("Should determine CSGD as eligible when financial need is at least $1 and total family income is under the threshold and eligible dependents are greater than 1.", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedFulltimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataTaxReturnIncome = 32999;
+    // Creates 4 eligible and 4 not eligible dependents.
+    assessmentConsolidatedData.studentDataDependants = [
+      createFakeStudentDependentEligible(
+        DependentEligibility.Eligible0To18YearsOld,
+      ),
+      createFakeStudentDependentEligible(
+        DependentEligibility.Eligible18To22YearsOldAttendingHighSchool,
+      ),
+      createFakeStudentDependentEligible(
+        DependentEligibility.Eligible18To22YearsOldDeclaredOnTaxes,
+      ),
+      createFakeStudentDependentEligible(
+        DependentEligibility.EligibleOver22YearsOld,
+      ),
+      createFakeStudentDependentNotEligible(
+        DependentEligibility.Eligible0To18YearsOld,
+      ),
+      createFakeStudentDependentNotEligible(
+        DependentEligibility.Eligible18To22YearsOldAttendingHighSchool,
+      ),
+      createFakeStudentDependentNotEligible(
+        DependentEligibility.Eligible18To22YearsOldDeclaredOnTaxes,
+      ),
+      createFakeStudentDependentNotEligible(
+        DependentEligibility.EligibleOver22YearsOld,
+      ),
+    ];
+    // Act
+    const calculatedAssessment = await executeFulltimeAssessmentForProgramYear(
+      PROGRAM_YEAR,
+      assessmentConsolidatedData,
+    );
+    // Assert
+    // calculatedDataTotalFamilyIncome must be below the threshold for
```

Comment:
how much is the threshold, just for my info

Reply:
Thresholds are kept on DMNs and they have all sorts of use. They look like the one below.
![image](https://github.com/bcgov/SIMS/assets/61259237/f2cb5b52-a8cf-4408-b8f6-89a382119127)